script.js

```diff
@@ -1,12 +1,424 @@
+const mySearchInput = document.querySelector("#search");
+const myDisplay = document.querySelector(".describe");
+const selectMovie = document.getElementById("movies");
+const mySelectTag = document.querySelector("#episode");
+const allEps = document.createElement("option") 
+allEps.innerHTML = "All Episodes"
+const allEpsAtr = document.createAttribute("id")
+allEpsAtr.value = 0
+allEps.setAttributeNode(allEpsAtr)
+mySelectTag.appendChild(allEps)
+const allMovies = document.createElement("option")
+allMovies.innerHTML = "Choose your Movie"
+const allMoviesAtr = document.createAttribute("id")
+allMoviesAtr.value = null
+allMovies.setAttributeNode(allMoviesAtr)
+selectMovie.appendChild(allMovies)
 
+// making page for Episodes
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  episodeList.forEach((element) => {
+    const divResponsive = document.createElement("div");
+    const divCard = document.createElement("div");
+    const divBodyCard = document.createElement("div");
+    const divCardHeader = document.createElement("div");
+    divResponsive.className += "col-sm-12 col-md-4 mb-sm-3 mb-md-3 mb-lg-3 p-2";
+    divCard.className += "card col-12  p-sm-1";
+    divBodyCard.className += "card-body  col-md-12 p-sm-1";
+    const myHeader = document.createElement("h5");
+    myHeader.className +=
+      "card-title border col-md-12 col-lg-12  p-sm-4 shadow p-sm-3 mb-5  text-light";
+    myHeader.textContent = `${element.name}-S${element.season < 10 ? 0 : ""}${
+      element.season
+    }E${element.number < 10 ? 0 : ""}${element.number}`;
+    const myImage = document.createElement("img");
+    myImage.classList.add("card-img-top");
+    if(element.image != null){
+      myImage.src = element.image.medium;}
+    // myImage.src = element.image.medium;
+    const myText = document.createElement("p");
+    const fontAwesome = document.createElement("i");
+    fontAwesome.className += "fas fa-film ";
+    myText.className += "card-text pt-md-3 pt-lg-4 ";
+    myText.appendChild(fontAwesome).innerHTML =
+      "This Episode's Summary is :" + element.summary;
+    
+    if(element.image == null){
+        myImage.src = "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRZz41AfUN6JDKErhgZAmfX0V-xWuXwzuZ_e4zB64aELfACFS1n&usqp=CAU"
+        myImage.className += "myNullImage"
+        const myText = document.createElement("p");
+        const fontAwesome = document.createElement("i");
+        fontAwesome.className += "fas fa-film ";
+        myText.className += "card-text pt-md-3 pt-lg-4 ";
+        if(element.summary != ""){
+        myText.appendChild(fontAwesome).innerHTML =
+          "This Episode's Summary is :" + element.summary;
+        }
+        if(element.summary == ""){
+          myText.appendChild(fontAwesome).innerHTML =
+          "This Episode doesn't have any summary"
+        }
```

Comment:
Ellie, your code seems to huge amount of chunk to be dealt with. Just as a quick advice, maybe you can indent them to make them more readable for other developers. Also to comment at the beginning of each part increase the readibility.